Yank/tests/test_repex.py

```diff
@@ -678,13 +678,16 @@ def test_repex_create(self):
                 assert isinstance(restored_move, mmtools.mcmc.LangevinDynamicsMove)
 
             # Options have been stored.
-            option_names, _, _, defaults = inspect.getargspec(repex.__init__)
-            option_names = option_names[2:]  # Discard 'self' and 'mcmc_moves' arguments.
-            defaults = defaults[1:]  # Discard 'mcmc_moves' default.
-            options = reporter.read_dict('options')
-            assert len(options) == len(defaults)
-            for key, value in zip(option_names, defaults):
-                assert options[key] == value
+            stored_options = reporter.read_dict('options')
+            options_to_store = dict()
+            for cls in inspect.getmro(type(repex)):
+                parameter_names, _, _, defaults = inspect.getargspec(cls.__init__)
```

Comment:
`getfullargspec` here, especially because the `Repex` class is a `kwarg` only class